src/components/display/DisplayImage.js

```diff
@@ -2,60 +2,63 @@ import axiosInstance from "../api/axios";
 import { useState, useEffect } from "react";
 import "./DisplayImage.css";
 
-const DisplayImage = ({ images, setImages }) => {
- 
-  // const [showMore, setShowMore] = useState(5);
+const DisplayImage = ({ images, setImages, err, setErr }) => {
+  const [perPage, setPerPage] = useState("");
 
   useEffect(() => {
     const showImage = async () => {
       try {
-        const res = await axiosInstance.get();
-        setImages(res.data.results);
-        console.log(res);
-      } catch (error) {
-        console.log(error);
+        const { data } = await axiosInstance.get(`?&offset=${perPage}`);
+        console.log(data)
+        setImages(data.results);
+        if(!data.next){
+          setErr("No more images")
+        }
+      } catch {
+        setErr("Error showing images");
       }
     };
     showImage();
-  }, []);
+  }, [setImages, perPage, setErr]);
+
+  const loadMore = () => {
+    setPerPage((more) => more + 1); //not perfect yet
```

Comment:
`more` is not a good name ... I went looking for where `more` was created.

And `perPage` is a string, and you are trying to add a number to it

Reply:
I will change this to page

Reply:
I am still working on getting the paginated part rightly from the api cause the 1 added loading more for me presently but not checking through the entire images as I preferred